JetMETCorrections/Modules/interface/CorrectedJetProducer.h

```diff
@@ -60,15 +57,17 @@ namespace reco {
   }
 
   template<class T>
-  void CorrectedJetProducer<T>::produce(edm::Event& fEvent,
-					 const edm::EventSetup& fSetup)
+  void CorrectedJetProducer<T>::produce(edm::StreamID, edm::Event& fEvent, const edm::EventSetup& fSetup) const
   {
+    // cache - note that variable length arrays are a GCC extension
+    reco::JetCorrector const * correctors[mCorrectorTokens.size()];
```

Comment:
Please don't use this construct since it is not in the C++ standard. Using a std::vector<> here is not going to even show up on the performance I bet.